app/views/static_pages/help.html.erb

```diff
@@ -1,12 +1,13 @@
-<%= provide :title, "Help" %>
-
+<% provide :title, "Help" %>
 <h1>Help</h1>
 <p>
   Get help on the Ruby on Rails Tutorial at the
-  <a href="http://www.railstutorial.org/help">Rails Tutorial
-  help page</a>.
-  To get help on this sample app, see the
-  <a href="http://www.railstutorial.org/book"><em>Ruby on Rails
-  Tutorial</em>
-  book</a>.
+  <a
```

Comment:
because you already use `link_to` tag helper => remove `a` tag
refactoring from line 5 -> 12
```suggestion
   <%= link_to "Rails tutorial help", "http://www.railstutorial.org/help" %>
   To get help on this sample app, see the
   <%= link_to  "http://www.railstutorial.org/book" do %>
     <em>Ruby on Rails Tutorial</em>
   <% end %>
```